### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by Leerw** [wangyida/softpool#15]
Could you please provide the pretrained model for ShapeNet at the resolution of 16384? I noticed that the checkpoint is large, so, is it possible to share it on Google-Drive or, whatever else can satisfy the storage requirement?

**Comment by 3dpose** [comment]
Yes, can you please share the pretrained model for ShapeNet at the resolution of 16384? We'd like to use it to perform quantitative comparison.
